src/src/tx_main.cpp

```diff
@@ -719,6 +719,7 @@ void HandleUpdateParameter()
 void ICACHE_RAM_ATTR ModelUpdateReq()
 {
   UpdateModelReq = true;
+  config.SetModelId(crsf.getModelID());
```

Comment:
I think we should flip these 2 lines around though

Reply:
Yup I would agree that would be the safer way to arrange them, but I had a few lockups on boot and I realized it was possible the model pointer wouldn't have been initialized unless we had received the modelchange message. I moved the set to `ChangeRadioParams()` which is both called in `setup()` and when `ModelUpdateReq()` fires, so it is always initialized properly now and can't have any races because it all happens on the main loop,

Should kill both birds with one stone and no extra code space, right?